lunatrace/bsl/frontend/src/pages/project/builds/VulnerablePackageListItem/VulnerablePackageCardHeader.tsx

```diff
@@ -41,20 +50,25 @@ export const VulnerablePackageCardHeader: React.FunctionComponent<VulnerablePack
           </Col>
           <Col sm={{ span: 6 }}>
             <div style={{ float: 'right', textAlign: 'right' }}>
-              <Card.Title>
-                <span className="text-right darker"> Severity: </span>
-                <div style={{ display: 'inline-block' }} className="vulnerability-severity-badge">
-                  <h4 className={`p-1 ${pkg.severity}`} style={{ display: 'inline' }}>
-                    {pkg.severity}
-                  </h4>
-                </div>
-              </Card.Title>
-              {pkg.cvss_score ? (
-                <Card.Subtitle>
-                  {' '}
-                  <span className="darker">CVSS: </span>
-                  {pkg.cvss_score}
-                </Card.Subtitle>
+              {mostSevereSeverity ? (
```

Comment:
I'm trying to figure out why we are only showing a card for the most vulnerable one and what the flow of this logic is